versions.json

```diff
@@ -0,0 +1,123 @@
+{
+  "design-system": [
+    "6.0.2",
```

Comment:
Note that it did what I expected here. After I bumped everything to `x.0.2`, it added them to the tops of these lists